applications/StructuralMechanicsApplication/tests/structural_response_function_test_factory.py

```diff
@@ -48,7 +48,7 @@ def setUp(self):
             self.problem_name = parameters["problem_data"]["problem_name"].GetString()
 
             model = KratosMultiphysics.Model()
-            self.response_function = structural_response_function_factory.CreateResponseFunction("dummy", parameters["kratos_response_settings"], model)
+            self.response_function = structural_response_function_factory.CreateResponseFunction("dummy", parameters["response_settings"], model)
```

Comment:
@MFusseder can you have a look if you need the backward-compatibility here? If so i can add a deprecation warning for `kratos_response_settings`.

Reply:
No, this is not necessary. I will simply adjust the settings of my examples after this changes entered the master.